addons/mail/wizard/mail_compose_message.py

```diff
@@ -1081,7 +1083,7 @@ def _prepare_mail_values_dynamic(self, res_ids):
         if not self.reply_to_force_new:
             # compute alias-based reply-to in batch
             reply_to_values = RecordsModel.browse(res_ids)._notify_get_reply_to(default=False)
-        if self.reply_to_force_new:
+        if self.reply_to_force_new or self.reply_to:
```

Comment:
Is this change still needed now that `reply_to_mode` is recomputed when `reply_to` is set?
I see that `reply_to_force_new` is updated in `inverse_reply_to_mode`